platform/solr/solr-factory-impl/src/main/java/org/codice/solr/factory/impl/Configsets.java

```diff
@@ -39,15 +44,14 @@
               "solr.xml",
               "solrconfig.xml",
               "stopwords.txt",
-              "stopwords_en.txt",
               "synonyms.txt"));
 
   private String configsetsPath;
 
   private Path defaultPath;
 
   public Configsets() {
-    this(Paths.get(System.getProperty("ddf.home", ""), "etc/solr/configsets"));
+    this(Paths.get(System.getProperty("ddf.home", ""), "etc", "solr", "configsets"));
```

Comment:
Yes, I believe this is the 'best practice' way to use `Paths.get` since it will insert the separator automatically, saving you from needing to hardcode it.